stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>Could you corroborate this value with something external, to validate your cleaning process?</p><p> </p><p>Looking at the match stats on the <a href="https://overwatchleague.com/en-us/statslab?statslab=matches" target="_blank">OWL site</a> (Team Map Stats, Team (All), Season (2021), vs. Opponent (All), Stage (All)) the Total Matches seems to be 874. I'm probably picking out the wrong numbers, but still curious about the discrepancy if there is one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>That's an excellent idea to check. The discrepancy of 874 vs 894 comes from the 874 not including draws. The OWL site says that 874 times someone lost, 874 times someone won, and 40 times someone drew, which means that there were 20 draws, since a single draw is counted for both teams. This matches with 894-874=20.</p>